### PR TITLE
add update config for remaining packages

### PR DESCRIPTION
**Pull request by rawlingsj**

### CODE REVIEW

File: cmake.yaml [hunk 1]
```diff
@@ -45,6 +45,7 @@ pipeline:
       DESTDIR="${{targets.destdir}}" ninja install
   - uses: strip
 update:
-  enabled: false
+  enabled: true
+  manual: true # be careful upgrading Make as it is a core package
```
Comment (luhring) [inline cmake.yaml:49]: nit:
```suggestion
  manual: true # be careful upgrading cmake as it is a core package
```

**Review by luhring — DISMISSED** [review]
So cool... And the explanations in the comments are super useful 💯

**Comment by rawlingsj** [comment]
a different CI failure
```
error getting package dependencies: could not find package libbsd=${{package.version}}-${{package.epoch}} in indexes
```

**Comment by rawlingsj** [comment]
Looks like this https://github.com/wolfi-dev/os/blob/bcf13d9e18be38be8d4e84d9864da00b4621aa26/libbsd.yaml#L33

maybe that's not needed

**Comment by kaniini** [comment]
Yeah we can get rid of that libbsd line, Melange already solves those kinds of dep relationships automatically :)

**Comment by rawlingsj** [comment]
green! just need a LGTM

**Review by kaniini — APPROVED** [review]

**Pull request merged**